02/styles/global.css

```diff
@@ -4,23 +4,48 @@
     box-sizing: border-box;
 }
 
+body {
+    display: grid;
+    grid-template-areas: 
+    "A"
+    "B"
+    "C";
+}
+
+
 .a, .b, .c {
     border: 3px solid transparent;
-    min-height: 50px;
+    height: 50vh;
 
     display: flex;
     justify-content: center;
     align-items: center;
+    
 }
 
 .a {
-    border-color: #ffffba
+    border-color: #ffffba;
+    grid-area: A;
 }
 
 .b {
     border-color: #ffdfba;
+    grid-area: B;
 }
 
 .c {
     border-color: #baffc9;
-}
\ No newline at end of file
+    grid-area: C;
+}
+
+@media (min-width: 600px) and (max-width: 768px) {
```

Comment:
W zadaniu było, że "jak przekroczy 600" - nie ma więc granicy górnej. 
Dlatego `and (max-width: 768px)` do usunięcia ;)